temp/prometheus-builder/README.md

```diff
@@ -22,5 +22,6 @@ scrape_configs:
     static_configs:
       - targets: ['localhost:9090']" > config/prometheus.yaml
 
+docker build -t sipian/prometheus-builder:v1.0.0 .
```

Comment:
I meant making it more generic by not including your repo name in the docker path.

Reply:
if you want to include the name than make is as 
`promettheus/prometheus-builder`

if we are to keep this than it would end up in the official prometheus repo.

Reply:
Oh my bad :stuck_out_tongue:

Reply:
I think just `prometheus-builder` is fine for now